File: 01/p1.py
with open("input", "r") as f:
    lines = f.readlines()
total = 0
position = 50
for line in lines:
    if line[0] == "L":
        position -= int(line.strip()[1:])
    elif line[0] == "R":
        position += int(line.strip()[1:])
    position = position % 100 
    if position == 0:
        total += 1
print(total)

File: 01/p1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main() {
    ifstream f("input", ifstream::in);
    string line;

    int position = 50;
    int total = 0;
    while (f >> line) {
        if (line.at(0) == 'L') {
            position -= stoi(line.substr(1));
        } else if(line.at(0) == 'R') {
            position += stoi(line.substr(1));
        }
        position = position % 100;
        if (position == 0)
            total += 1;
    }
    cout << total << endl;
    return 0;
}